yellowbrick/classifier/prcurve.py

```diff
@@ -383,6 +408,10 @@ def finalize(self):
         self.ax.set_ylabel("Precision")
         self.ax.set_xlabel("Recall")
 
+        if self.per_class:
+            self.ax.grid(False)
+
```

Comment:
Awesome! This looks so much cleaner &mdash; I'm actually wondering if we should turn off the grid for `per_class=False` also? Do you think it really adds anything?

Reply:
We can turn off the grid as it is not necessary in my opinion.